web_widget_digitized_signature/views/web_digital_sign_view.xml

```diff
@@ -1,9 +1,11 @@
-<?xml version="1.0" encoding="UTF-8"?>
 <odoo>
 
-    <template id="web_widget_digitized_signature_backend" name="web_widget_digitized_signature assets" inherit_id="web.assets_backend">
+    <template id="web_widget_digitized_signature_backend"
```

Comment:
No automatic syntax.

Reply:
@pedrobaeza Same thing. Migration is a good opportunity to make this file easily reviewable.

Reply:
The same answers: I don't see this as an improvement and any guideline says about that, and also it mustn't be included in the migration commit.